nncf/experimental/torch/sparsity/movement/algo.py

```diff
@@ -116,6 +115,24 @@ def _build_controller(self, model: NNCFNetwork) -> PTCompressionAlgorithmControl
         return MovementSparsityController(model, self._sparsified_module_info, self.config)
 
 
+
+MODEL_FAMILIES = ['bert', 'wav2vec2', 'swin']
+
+def is_supported_model_family(model: NNCFNetwork) -> None:
+    """
+    Checks whether the model family is supported by movement sparsity to conduct structured masking.
+
+    :param model: The compressed model wrapped by NNCF.
+    """
+    model_pymodules = inspect.getmodule(model).__name__.split('.')
+    is_supported = False
+    if len(model_pymodules) >= 3 and model_pymodules[:2] == ['transformers', 'models']:
+        # the case of input model defined by HuggingFace's transformers
+        model_family = model_pymodules[2]
+        if model_family in MODEL_FAMILIES:
+            is_supported = True
+    return is_supported
```

Comment:
I'd personally go with my original suggestion as it's shorter, but this is fine too

Reply:
ok, saved one more line 😆